controllers/merge/merge.go

```diff
@@ -39,6 +39,8 @@ func RetainClusterObjectFields(desiredObj, clusterObj *unstructured.Unstructured
 		fallthrough
 	case util.MutatingWebhookConfigurationKind:
 		return retainWebhookConfigurationFields(desiredObj, clusterObj)
+	case util.SecretKind:
```

Comment:
Doh, good catch!

Reply:
@mmirecki I'm actually thinking that this change may be all that was needed to make #149 work successfully. WDYT?